### PR TITLE
prevent zero networkID 

### PR DESCRIPTION
**Pull request by seunlanlege**

### CODE REVIEW

**Comment by ordian** [comment]
looks like some tests need to be fixed

**Comment by 5chdn** [comment]
Hold on? This pull request should not have been merged.

1. It does not explain **why** network ID can't be zero
2. The network ID of Olympic actually _is_ zero, changing it basically creates a new network.

File: ethcore/res/ethereum/olympic.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		"accountStartNonce": "0x00",
 		"maximumExtraDataSize": "0x0400",
 		"minGasLimit": "125000",
-		"networkID" : "0x0",
+		"networkID" : "0xf0",
```
Comment (5chdn) [inline ethcore/res/ethereum/olympic.json]: Should be reverted

Comment (sorpaas) [reply ethcore/res/ethereum/olympic.json]: Yeah sorry I missed this. This line should indeed be reverted and fixed (or we may just consider removing olympics support).

**Comment by 5chdn** [comment]
Ok, so we might want to remove Olympic completely

**Comment by seunlanlege** [comment]
alright will create an issue

**Pull request merged**